common/laws/00_monarchy.txt

```diff
@@ -62,13 +62,12 @@ monarchy_military_reforms = {
 	
 	relaxed_recruiting_standards = { #Noble Retinues - Macedon
 		modifier = {
-			levy_size_multiplier = 0.05
-			happiness_for_same_culture_modifier = 0.05
-			happiness_for_wrong_culture_modifier = -0.05
+			levy_size_multiplier = 0.025
```

Comment:
so i think these numbers were getting a bit out of control. i made some effort to reduce them across the board in these laws and normalize them a bit.